webpack/assets/javascripts/react_app/components/HostsIndex/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { CubeIcon } from '@patternfly/react-icons';
+import { translate as __ } from '../../common/I18n';
+import TableIndexPage from '../PF4/TableIndexPage/TableIndexPage';
+import { HOSTS_API_PATH, API_REQUEST_KEY } from '../../routes/Hosts/constants';
+
+const HostsIndex = () => {
+  const columns = {
+    name: {
+      title: __('Name'),
+      wrapper: ({ can_edit: canEdit, id, name }) =>
+        canEdit ? (
+          <a href={`/models/${id}/edit`}>{name}</a>
+        ) : (
+          <span>{name}</span>
+        ),
+      isSorted: true,
+    },
+  };
+
+  const computeContentSource = search =>
+    `/change_host_content_source?search=${search}`;
+
+  const customActionKebabs = [
+    {
+      title: __('Change content source'),
```

Comment:
This should actually be in Katello since it's linking to a Katello page.

Reply:
Should set this up similar to the way it's already done for host details.

See -> https://github.com/Katello/katello/blob/a57828483396b6e02e4b6d849202a0253abe9004/webpack/components/extensions/HostDetails/ActionsBar/index.js#L71

Exported from Katello to Foreman here -> https://github.com/Katello/katello/blob/a57828483396b6e02e4b6d849202a0253abe9004/webpack/global_index.js#L70

Consumed here in Foreman -> 
https://github.com/theforeman/foreman/blob/d329c933388647d3260684d0e3d9bbc9a34397b4/webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/Selectors.js#L6

which is then rendered here -> https://github.com/theforeman/foreman/blob/d329c933388647d3260684d0e3d9bbc9a34397b4/webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js#L49

Reply:
Fixed!